plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?eclipse version="3.4"?>
+<?eclipse version="3.2"?>
```

Comment:
@elharo That is the dtd version, it has not much correlation with the Eclipse versions, for Mars it is 3.2:
http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fplugin_manifest.html